<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="batchGroupBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>120</y>
      <width>440</width>
      <height>355</height>
     </rect>
    </property>
    <property name="title">
     <string>Batch Watermarking</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>420</width>
       <height>315</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>-1</number>
      </property>
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="label_10">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>21</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Output Folder:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QRadioButton" name="copyFilenameRadioBtn">
        <property name="text">
         <string>Copy Filename</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_9">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>21</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Source Folder:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="outDirTextBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="quickOutDirBtn">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectOutDirBtn">
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Select Folder...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="9" column="1">
       <widget class="QRadioButton" name="addWatermarkedSuffixRadioBtn">
        <property name="text">
         <string>Add &quot;watermarked&quot; suffix</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="addSuffixRadioBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>21</height>
           </size>
          </property>
          <property name="text">
           <string>Add suffix</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="suffixTextBox">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>21</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="srcDirTextBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="quickSrcDirBtn">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectSrcDirBtn">
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Select Folder...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLineEdit" name="batchWatermarkTextBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="addPrefixRadioBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>21</height>
           </size>
          </property>
          <property name="text">
           <string>Add suffix</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="prefixTextBox">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>21</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="12" column="0" colspan="2">
       <widget class="QProgressBar" name="batchProgressBar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>10</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>21</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Watermark Text:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QPushButton" name="goBtn">
        <property name="text">
         <string>Mark PDFs</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="batchGroupBox_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>440</width>
      <height>120</height>
     </rect>
    </property>
    <property name="title">
     <string>Single Document</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>420</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>-1</number>
      </property>
      <item row="3" column="0" colspan="2">
       <widget class="QLineEdit" name="singleWatermarkTextBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_13">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>21</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Watermark Text:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="singleGoBtn">
        <property name="text">
         <string>Mark Single PDF</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <action name="actionSelect_File">
   <property name="text">
    <string>Select File...</string>
   </property>
  </action>
  <action name="actionSelect_Folder">
   <property name="text">
    <string>Select Folder...</string>
   </property>
  </action>
  <action name="actionSelect_Output_Folder">
   <property name="text">
    <string>Select Output Folder...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
